stable/wordpress/templates/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+ name: {{ template "fullname" . }}
+ labels:
```

Comment:
I would add support for custom annotations, similar to https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/alertmanager-ingress.yaml#L7, since Ingress controllers rely on annotations quite a bit.